sdk/ai/azopenai/example_client_getchatcompletions_test.go

```diff
@@ -78,7 +78,10 @@ func ExampleClient_GetChatCompletions() {
 
 	for _, choice := range resp.Choices {
 		gotReply = true
-		fmt.Fprintf(os.Stderr, "Content[%d]: %s\n", *choice.Index, *choice.Message.Content)
+
+		if choice.Message != nil && choice.Message.Content != nil {
```

Comment:
Would it be possible to add a comment here explaining when a response might contain only filtering info.

Reply:
I've adjusted it. The current behavior isn't ideal so it's best to not assume it's one or the other. I did change it so it's clear what data is what, however.